### PR TITLE
Added Amicable number calculator with steps

### LINKED ISSUE #3200: Amicable Number
A calculator to check if number is amicable or not.

I'd like to work on this.

### PR DESCRIPTION
**Pull request by Khushi-agarwal**
## Related Issue

- Info about Issue or bug

Fixes #3200 

#### Describe the changes you've made

A clear and concise description of what you have done to successfully close your assigned issue. Any new files? or anything you feel to let us know!

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Review by sairish2001 — CHANGES_REQUESTED** [review]
Have a look to other Calculator files,
You need to use only HTML,CSS JavaScript. 
In Calculators/amicable.html

you put unnecessary C++ code, mistakenly.
Have a look and design UI accordingly.

I think you are not checking the work you are doing by running live server locally!
Are you?

**Comment by Khushi-agarwal** [comment]
> Have a look to other Calculator files,
> You need to use only HTML,CSS JavaScript.
> In Calculators/amicable.html
> 
> you put unnecessary C++ code, mistakenly.
> Have a look and design UI accordingly.
> 
> I think you are not checking the work you are doing by running live server locally!
> Are you?

Yes that was mistakenly added there.It was a different file but I do not know how come it got added there maybe due to the same name.I apologize for it and have corrected it. @sairish2001

File: calculators/amicable.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+		 renderMathInElement(document.getElementById('outputsteps'));
+		 renderMathInElement(document.getElementById('outputresult'));
+	 }
+  </script>
+ <div>
+	 <input type="number" id="num1amicable" class="form__field">
+	 <input type="number" id="num2amicable" class="form__field">
+	
+	 
+	 <button type="button" class="btn btn-outline-light" onclick="callamicables() ">Solve</button>
```
[the remark this answers is not in the document]
Comment (Khushi-agarwal) [reply calculators/amicable.html]: Resolved

**Review by sairish2001 — APPROVED** [review]
Great Work:)

**Pull request merged**